app/controllers/organizations_controller.rb

```diff
@@ -1,17 +1,22 @@
+# rubocop:disable ClassLength
```

Comment:
Maybe it's time to start breaking some of this logic out into `lib` with service objects.